docs/content-services/services/category.service.md

```diff
@@ -50,6 +50,20 @@ Manages categories in Content Services.
     -   _categoryId:_ `string`  - The identifier of a category.
     -   _payload:_ `CategoryBody`  - Updated category body
     -   **Returns** [`Observable`](http://reactivex.io/documentation/observable.html)`<CategoryEntry>` - [`Observable`](http://reactivex.io/documentation/observable.html)&lt;CategoryEntry>
+-   **getCategoryLinksForNode**(nodeId: `string`): [`Observable`](http://reactivex.io/documentation/observable.html)`<`[`CategoryPaging`](https://github.com/Alfresco/alfresco-js-api/blob/master/src/api/content-rest-api/docs/CategoryPaging.md)`>`<br/>
+    Searches categories by their name.
```

Comment:
Incorrect description, looks like description for searchCategories

Reply:
Right, fixed now